docker/docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
         --hiveconf hive.server2.thrift.bind.host=0.0.0.0 \
         --hiveconf derby.system.home=/opt/spark/data/derby \
         --master=local[2] \
-        --conf spark.driver.memory=2G \
+        --conf spark.driver.memory=3G \
```

Comment:
Is it intentional to increase the RAM amount here? If the RAM usage goes up, we may have the problem of starting up the app as reported previously.

Reply:
This compose file only applies specifically to the test spark instance used for integration testing.

Reply:
Got it, I thought this is `featurebyte-app`. 😅